examples/device/ep/tests/control/control.py

```
@@ -0,0 +1,489 @@
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+import nixl_ep  # noqa: E402
+
+from utils import rank_server, store_group  # noqa: E402
+from utils.utils import CudaTimer, stats, tcp_store_barrier  # noqa: E402
+
+TCP_STORE_PORT = 9999
+RANK_SERVER_PORT = 10000
+
+
+def handle_sigterm(signum, frame, rank_client):
```

Ruff (0.15.9)

[warning] Unused function argument: `frame`

(ARG001)